internal/services/manager_test.go

```diff
@@ -34,11 +36,13 @@ func TestNewManager(t *testing.T) {
 				t.Setenv("DBUS_SYSTEM_BUS_ADDRESS", tc.systemBusSocket)
 			}
 
-			_, err := services.NewManager(context.Background(), tc.cacheDir, nil)
+			m, err := services.NewManager(context.Background(), tc.cacheDir, nil)
 			if tc.wantErr {
 				require.Error(t, err, "NewManager should have returned an error, but did not")
 				return
 			}
+			defer func() { _ = m.Stop() }()
```

Comment:
in that case, check that we don’t return an error as we don’t :) (so on the defer, unfortunately). You did too much Rust to check for errors :p

Reply:
I was trying so hard to use the `?` operator :smiling_face_with_tear:

Reply:
implicit error checking acting like exception? How dare you :)